sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -35,29 +35,29 @@
     private final Logger logger;
 
     /*
-     * Indicate that log level is at verbose level.
+     * Indicates that log level is at verbose level.
      */
-    private static final int VERBOSE_LEVEL = 1;
+    public static final int VERBOSE_LEVEL = 1;
```

Comment:
@alzimmermsft @JonathanGiles 
One thing I wanted to ask if there were negative repercussions to making this public. From an external user's point of view, it's hard to know what the log levels are without going to the source.

I have a link in:

Reply:
Yeah, ideally we wouldn't do this

Reply:
How do external customers decide which integer to set AZURE_LOG_LEVEL if we move this back to being private? If I set AZURE_LOG_LEVEL to 10, what behaviour should expect.